contributing-guides/style-guide.md

```diff
@@ -34,10 +34,10 @@ tldrl -f {{page.md}}
 For other ways to use `tldrl`, such as linting an entire directory, check out (what else!)
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldrl.md)
 
-If you're using the Node.js client of tldr-pages, you can preview a page locally using the `-f` flag (aka `--render`):
+If you're using the Node.js or Python clients of tldr-pages, you can preview a page locally using the `--render` argument (aka `-f` in Node.js, and `-r` in Python):
```

Comment:
```suggestion
If you're using the Node.js or Python client of tldr-pages, you can preview a page locally using the `--render` argument (aka `-f` in Node.js, and `-r` in Python):
```

Reply:
Another suggestion?

```suggestion
If you're using the Node.js or Python tldr-pages clients, you can preview a page locally using the `--render` argument (aka `-f` in Node.js, and `-r` in Python):
```

Reply:
Sounds good, however it would still be `client` right?

Reply:
`clients` should be plural, because you're referring to multiple clients theere - both the Node.js and Python clients.

Reply:
@sbrl should I add the C client to the list? or maybe a generalization like `try --render in your client.`

Reply:
I'd write this line like this:
```suggestion
Your client may be able to preview a page locally using the `--render` flag:
```
That way,
- We don't need to list the clients
- We don't need to list the possible short options

Reply:
Sounds like a good option, especially since most users just install what's in the repos and don't know which one they're using.